diff --git a/src/vs/platform/windows/electron-main/windowImpl.ts b/src/vs/platform/windows/electron-main/windowImpl.ts
index 089c19b..486c8a0 100644
--- a/src/vs/platform/windows/electron-main/windowImpl.ts
+++ b/src/vs/platform/windows/electron-main/windowImpl.ts
@@ -230,7 +230,8 @@ export class CodeWindow extends Disposable implements ICodeWindow {
                                        // Enable experimental css highlight api https://chromestatus.com/feature/5436441440026624
                                        // Refs https://github.com/microsoft/vscode/issues/140098
                                        enableBlinkFeatures: 'HighlightAPI',
-                                       sandbox: true
+                                       nodeIntegration: true,
+                                       contextIsolation: false
                                },
                                experimentalDarkMode: true
                        };
diff --git a/src/main.js b/src/main.js
index d986fee..d570bb4 100644
--- a/src/main.js
+++ b/src/main.js
@@ -27,9 +27,6 @@ const { getUNCHost, addUNCHostToAllowlist } = require('./vs/base/node/unc');
 const product = require('../product.json');
 const { app, protocol, crashReporter, Menu } = require('electron');

-// Enable sandbox globally
-app.enableSandbox();
-
 // Enable portable support
 const portable = bootstrapNode.configurePortable(product);

